website_seo_sale/__openerp__.py

```diff
@@ -0,0 +1,22 @@
+{
+    'name': 'eCommerce SEO Optimized',
+    'category': 'Website',
+    'sequence': 55,
+    'summary': 'Sell Your Products Online',
+    'website': 'http://www.bloopark.de',
+    'version': '1.0',
+    'description': """
+OpenERP E-Commerce SEO Optimized
+==================
+
+        """,
+    'depends': [ 'website_sale',],
+    'data': [
+        'views/templates.xml',
+    ],
+    'demo': [
+    ],
+    'qweb': ['static/src/xml/*.xml'],
```

Comment:
Maybe is not necessary the 'qweb' key if you actually don't have any template to load.